tests/output_parsers/test_selection.py

```diff
@@ -26,3 +27,19 @@ def test_parse(output_parser: SelectionOutputParser) -> None:
     assert len(parsed.parsed_output) == 2
     assert parsed.parsed_output[0].choice == 1
     assert parsed.parsed_output[0].reason == "just because"
+
+
+def test_bugfix_llm_output_with_string_outside_of_json_issue_3135(
+    output_parser: SelectionOutputParser,
+) -> None:
+    # https://github.com/jerryjliu/llama_index/issues/3135
+    output = """
+    Output:
+    [
+      {
+        "choice": 1,
+        "reason": "Useful for questions that seek to gain a "
+      }
+    ]
+    """
+    output_parser.parse(output)
```

Comment:
can we assert that this matches the expected dict?

Reply:
Thanks for all the feedback!